.github/workflows/instructlab_baseimages_build_push.yaml

```diff
@@ -102,15 +95,84 @@ jobs:
       
       - name: Build Image
         id: build_image
-        uses: redhat-actions/buildah-build@v2.13
+        run: make image DRIVER_VERSION=${{ matrix.driver_version }} ARCH=${{ matrix.archs }}
+        working-directory: ${{ matrix.context }}
+        
+      - name: Login to Container Registry
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: redhat-actions/podman-login@v1.7
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.REGISTRY_USER }}
+          password: ${{ secrets.REGISTRY_PASSWORD }}
+
+      - name: Push image
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: redhat-actions/push-to-registry@v2.8
         with:
-          image: ${{ env.REGISTRY_ORG }}/${{ matrix.image_name }}
-          platforms: ${{ matrix.platforms }}
-          labels: |
-            ${{ matrix.label }}
-          tags: latest
-          containerfiles: ${{ matrix.containerfile }}
-          context: ${{ matrix.context }}
+          image: ${{ steps.build_image.outputs.image }}
+          tags: ${{ steps.build_image.outputs.tags }}
+          registry: ${{ env.REGISTRY }}
+
+  intel-bootc-image:
```

Comment:
This and the next job looks very similar to the first one. Is it possible to consolidate that with the matrix section?

Reply:
We could consolidate jobs 3 and 4, this change is coming